FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h

```diff
@@ -54,6 +54,11 @@ NS_SWIFT_NAME(AppCheck)
 /// Requests Firebase app check token. This method should *only* be used if you need to authorize
 /// requests to a non-Firebase backend. Requests to Firebase backend are authorized automatically if
 /// configured.
+///
+/// If your non-Firebase backend exposes sensitive or expensive endpoints that have low traffic
+/// volume, consider protecting it with [Replay
+/// Protection](https://firebase.google.com/docs/app-check/custom-resource-backend#replay-protection).
+/// In this case, use the ``limitedUseTokenWithCompletion()`` instead to obtain a limited-use token.
```

Comment:
nit: Potentially use the Swift signature `limitedUseToken(completion:)` here too. (I'll ping our group chat for clarification on which to put in the docs and changelog)

Reply:
Sounds good to me, thanks.

Reply:
Checked with the team and the Swift signature is preferred since we're moving towards Swift-only public docs. Thanks for updating this!

Reply:
I have changed to limitedUseToken(completion:) here and in the changeLog too. Try to shoot for the cut tomorrow. If it turns out this need to be modified back to objc signature, I can fix it later. Is this okay for you? @andrewheard